File: .ProgramingBasics/SimpleCalculations/09.CelsiusToFahrenheit/src/CelsiusToFahrenheit.java
import java.util.Scanner;

public class CelsiusToFahrenheit {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double gradus = Double.parseDouble(scanner.nextLine());
        double celsius = gradus * 1.80000 + 32;

        System.out.println(celsius);
    }
}

File: ProgramingBasics/SimpleCalculations/09.CelsiusToFahrenheit/09.CelsiusToFahrenheit/CelsiusToFahrenheit.cs
using System;

namespace _09.CelsiusToFahrenheit
{
    public class CelsiusToFahrenheit
    {
        public static void Main(string[] args)
        {
            var gradus = double.Parse(Console.ReadLine());
            var celsius = gradus * 1.80000 + 32;

            Console.WriteLine(celsius);
        }
    }
}
